### DISCUSSION TITLE
It's possible to do a "hover tooltip" showing some text in the json data?

### DISCUSSION THREAD

**Discussion by Ricardosgeral** [vakata/jstree#2667] [Q&A] [answered]
Hi all
Hi have a tree populated with a json that has, for each node, the fields "text" and "title" (which is a longer string than "text").

To make the tree more readable to the user, I would like to show the content of the "title" as a "hover tooltip", that is, when the user has the mouse over the node a hover appears with the value of the "title".

Can this be done with the JStree?

**Comment by ApiumJacob** [comment, marked as answer]
Sorry, here is the better code:

"a_attr": {"title": "stick to your day job"}
```
    $('#jstree_div').jstree({
        "core" : {
            "multiple" : true,
            "themes" : {
                "variant" : "medium",
            },
            data : [{"id": "1", "text": "Hello", "a_attr": {"title": "stick to your day job"}},
                    {"id": "2", "text": "World", "a_attr": {"title": "never take advise from a stranger"}}
                   ]
        },
        "plugins" : [ "sort" ] 
    });

```
